src/Frontend/Modules/Mailmotor/Layout/Widgets/Subscribe.html.twig

```diff
@@ -9,19 +9,20 @@
 
   {% block widget_body %}
     <div class="widget-body">
-      <form action="{{ geturlforblock('Mailmotor', 'Subscribe' ) }}" method="post" class="form-horizontal">
+      {{ form_start(form) }}
         <input type="hidden" name="form_token" id="formToken" value="{{ formToken }}" />
         <input type="hidden" name="form" value="subscribe" />
-        <label class="control-label" for="email">
-          {{ 'lbl.Email'|trans|ucfirst }}{{ macro.required }}
-        </label>
+        {% if form.interests %}
+          {{ form_row(form.interests) }}
+        {% endif %}
+        {{ form_label(form.email) }}
         <div class="input-group">
-          <input type="email" value="" id="email" name="email" class="form-control" placeholder="{{ 'lbl.YourEmail'|trans|ucfirst }}" />
+          {{ form_widget(form.email) }}
           <span class="input-group-btn">
-            <input id="send" class="btn btn-default" type="submit" name="send" value="{{ 'lbl.Subscribe'|trans }}" />
+            {{ form_widget(form.subscribe) }}
           </span>
         </div>
```

Comment:
Add form_rest, so if somebody updates the SubscribeType Form everything keeps working.

Reply:
@jeroendesloovere the form_rest will automatically be executed by the form_end